app/controllers/manuals_controller.rb

```diff
@@ -5,33 +5,49 @@ class ManualsController < ApplicationController
 
   before_action :ensure_manual_is_found
   before_action :ensure_document_is_found, only: :show
-  before_action :load_manual
   before_action :prevent_robots_from_indexing_hmrc_manuals
 
   def index
-    set_expiry(manual)
+    set_expiry(content_store_manual)
+
+    render :index, locals: {
+      presented_manual: presented_manual
+    }
   end
 
   def show
-    set_expiry(document)
-    @document = Document.new(document, @manual)
+    set_expiry(document_from_repository)
+    document = Document.new(document_from_repository, manual)
+
+    render :show, locals: {
+      presented_manual: presented_manual,
+      document: document
+    }
   end
 
   def updates
+    render :updates, locals: {
+      presented_manual: presented_manual
+    }
   end
 
 private
 
+  def presented_manual
+    @presented_manual ||=
+      ManualPresenter.new(manual: manual, view_context: view_context)
```

Comment:
Oh, yes, this is good.  I forgot that the controller has a `view_context` that we can use instead of relying on `self` in the templates.  :+1: